modules/installing-aws-load-balancer-operator.adoc

```diff
@@ -19,15 +17,14 @@ You can install the AWS Load Balancer Operator from the OperatorHub by using the
 . Navigate to *Operators* → *OperatorHub* in the {product-title} web console.
 . Select the *AWS Load Balancer Operator*. You can use the *Filter by keyword* text box or use the filter list to search for the AWS Load Balancer Operator from the list of Operators.
 . Select the `aws-load-balancer-operator` namespace.
-. Follow the instructions to prepare the Operator for installation.
-. On the *AWS Load Balancer Operator* page, click *Install*.
+. (Optional) Follow the instructions to prepare the Operator for installation in custom scenarios when you want to use a security token service (STS) or a User Provisioned Infrastructure (UPI).
```

Comment:
We have a separate chapter for the STS installation: "Installing the AWS Load Balancer Operator on a cluster using the AWS Security Token Service". Should we drop this step or reference the STS chapter?

Reply:
Are you talking about https://docs.openshift.com/container-platform/4.15/networking/aws_load_balancer_operator/installing-albo-sts-cluster.html ?

There is no install steps there at the moment, and we probably need to include these install steps at the end of the setup directions.

Reply:
> Are you talking about https://docs.openshift.com/container-platform/4.15/networking/aws_load_balancer_operator/installing-albo-sts-cluster.html ?

Yes.

> There is no install steps there at the moment, and we probably need to include these install steps at the end of the setup directions.

I would prefer a link just to avoid redundancy. But I don't know what your conventions about this.